pkg/queue/manager_test.go

```diff
@@ -708,21 +708,37 @@ func TestHeadsAsync(t *testing.T) {
 		t.Fatalf("Failed adding kueue scheme: %s", err)
 	}
 	now := time.Now().Truncate(time.Second)
-	cq := utiltesting.MakeClusterQueue("fooCq").Obj()
+	clusterQueues := []*kueue.ClusterQueue{
+		utiltesting.MakeClusterQueue("fooCq").Obj(),
+		utiltesting.MakeClusterQueue("barCq").Obj(),
+	}
 	wl := kueue.Workload{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Workload",
+			APIVersion: "kueue.x-k8s.io/v1alpha1",
+		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name:              "a",
 			CreationTimestamp: metav1.NewTime(now),
 		},
 		Spec: kueue.WorkloadSpec{QueueName: "foo"},
 	}
-	q := kueue.Queue{
-		ObjectMeta: metav1.ObjectMeta{Name: "foo"},
-		Spec: kueue.QueueSpec{
-			ClusterQueue: "fooCq",
+	var newWl kueue.Workload
+	queues := []kueue.Queue{
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "foo"},
+			Spec: kueue.QueueSpec{
+				ClusterQueue: "fooCq",
+			},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "bar"},
+			Spec: kueue.QueueSpec{
+				ClusterQueue: "barCq",
+			},
 		},
 	}
-	wantHeads := []workload.Info{
+	defaultHeads := []workload.Info{
```

Comment:
get rid of this variable and put it in the test table when necessary.

Reply:
No, I just mean to get rid of this variable and copy paste the list of workloads into the test table.